app/script_variant.py

```diff
@@ -31,9 +31,20 @@ def read_dictionary(file_name):
         return None
     return dictionary
 
+def combine_name_location_dictionary(name_dict, location_dict):
```

Comment:
How about putting a single file which both contains names and locations? It seems simpler.